src/main/java/cart/controller/CartControllerAdvice.java

```diff
@@ -0,0 +1,20 @@
+package cart.controller;
+
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class CartControllerAdvice {
+
+    @ExceptionHandler(IllegalArgumentException.class)
+    public ResponseEntity handle(IllegalArgumentException e) {
```

Comment:
제네릭을 사용할 때 raw 타입을 사용하는 것은 권장되지 않는 방법이에요. ResponseEntity를 사용하는 부분을 모두 아래처럼 수정해보면 어떨까요? ☺️
```suggestion
    public ResponseEntity<String> handle(IllegalArgumentException e) {
```

- https://rules.sonarsource.com/java/RSPEC-3740

Reply:
좋은 플러그인 알려줘서 고맙습니다 ㅎㅎ